Documentation/HowToGuides/UsingCombinedActions/README.md

```diff
@@ -0,0 +1,205 @@
+
+![RenameToVertical](assets/images/RenameToVertical.png)
+
+In the Axis Settings on the `Boolean To 1DAxis Action` Facade of the `Boolean To 1DAxis Action Vertical` GameObject, drag and drop the `Input.UnityInputManager.ButtonAction S` GameObject into the `Negative Input` parameter and drag and drop the `Input.UnityInputManager.ButtonAction W` GameObject into the `Positive Input` parameter.
+
+![Drag And Drop Into Positive And Negative Parameters](assets/images/DragAndDropIntoVerticalPositiveAndNegativeParameters.png)
+
+### Step 6
+
+Create a new `Empty` Unity Object by selecting `Main Menu -> GameObject -> Create Empty` and rename it to `Movement`. Click `Add Component` and add a `Transform Position Mutator` component.
+
+![Add Position Mutator](assets/images/AddPositionMutator.png)
+
+### Step 7
+
+Drag and drop the `Capsule` GameObject into the `Target` parameter on the `Transform Position Mutator` Facade.
+
+![Drag And Drop Capsule](assets/images/DragAndDropCapsule.png)
+
+### Step 8
+
+Expand the `Tilia.Input.CombinedActions.Unity` Package directory in the Unity Project window and select the `Packages -> Tilia.Input.CombinedActions.Unity -> Runtime -> Prefabs` directory then drag and drop the `Input.CombinedActions.AxesToVector3Action` prefab into the Unity hierarchy window.
+
+![Import Axes To Vector3 Action](assets/images/ImportAxesToVector3Action.png)
+
+### Step 9
+
+On the `Input.CombinedActions.AxesToVector3Action` GameObject, Drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction Horizontal` GameObject into the `Lateral Axis` property on the `Axes To Vector3` component.
+
+![Lateral Axis](assets/images/LateralAxis.png)
+
+and drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction Vertical` GameObject into the `Longitudinal Axis` property on the `Axes To Vector3` component.
+
+![Longitudinal Axis](assets/images/LongitudinalAxis.png)
+
+### Step 10
+
+Select the `Input.CombinedActions.AxesToVector3Action` GameObject from the Unity Hierarchy and click the `+` symbol in the bottom right corner of the `Value Changed` event parameter on the `Axes To Vector3` component.
+
+Drag and drop the `Movement` GameObject into the event listener box that appears on the `Value Changed` event parameter on the `Axes To Vector3` component that displays `None (Object)`.
+
+![Drag Movement Into Event Listener](assets/images/DragMovementIntoEventListener.png)
+
+Select a function to perform when the `Value Changed` event is emitted. For this example, select the `TransformPositionMutator` -> `DoIncrementProperty` function (be sure to select `Dynamic Vector3 - DoIncrementProperty` for this example).
+
+![Set Value Changed Function](assets/images/SetValueChangedFunction.png)
+
+> Play the Unity scene you should now be able to control the GameObject using the A, S, D, W keys.
```

Comment:
Change the key sequence from `A, S, D, W` to `W, A, S, D` as this is the actual defacto terminology for this key layout:

https://en.wikipedia.org/wiki/Arrow_keys#WASD_keys